peekingduck/cli.py

```diff
@@ -136,8 +136,21 @@ def init(custom_folder_name: str) -> None:
     help="""Modify node configs by wrapping desired configs in a JSON string.\n
          Example: --node_config '{"node_name": {"param_1": var_1}}'""",
 )
-def run(config_path: str, node_config: str) -> None:
+@click.option(
+    "--log_level",
+    default="info",
+    help=("""Modify log level {"error", "warning", "info", "debug"}"""),
```

Comment:
(Duplicate) Parentheses may not be required. And slight inconsistency to the actual range of acceptable values? `CRITICAL` is an allowed value in `log_level_settings`

Reply:
Removed parentheses and added CRITICAL